client/blocks/plan-storage/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import QueryMediaStorage from 'components/data/query-media-storage';
+import { getMediaStorage } from 'state/sites/media-storage/selectors';
+import {
+	getSitePlan,
+	getSiteSlug,
+	isJetpackSite
+} from 'state/sites/selectors';
+
+import PlanStorageBar from './bar';
+
+class PlanStorage extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		mediaStorage: PropTypes.object,
+		siteId: PropTypes.number.isRequired,
```

Comment:
On an empty cache the console might throw an error. How about we relax this, and instead render null when siteId is not available.